platform/ui/src/components/measurementTable/MeasurementTable.js

```diff
@@ -25,25 +25,27 @@ class MeasurementTable extends Component {
     t: PropTypes.func,
     saveFunction: PropTypes.func,
     onSaveComplete: PropTypes.func,
+    lesionTracker: PropTypes.bool,
```

Comment:
I would rather you duplicate this component entirely than add a bool that alters its behavior.

Either we have a generic measurement table with slots or other more common ways to extend the component for this use case, or we have an entirely different component that is composed of small, more atomic, shared generic elements.

Reply:
I'll pass in custom timepoints header and custom header for the table component, i think that would make it more generic for future use cases as well. What do you think?